test/kind/config.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Cluster
```

Comment:
Not sure if it makes sense, but might be able to use the kind config from vault-helm: https://github.com/hashicorp/vault-helm/blob/main/test/kind/config.yaml

It does have some extra bits, so not sure.

Reply:
Good point, the extra stuff is harmless and if it needs to diverge in the future, we can create a special config for vault-k8s kind at that point.